UI/ui_tutorial.csv

```diff
@@ -141,45 +141,45 @@ wh_tl_mp_000245_v#0,"""PSE"""
-wh_missionpass#0,"""Quests / Mission Pass"""
-wh_sgemshop#0,"""AC Menu / SG Shop"""
-wh_sgshop#0,"""AC Menu / Purchase Star Gems"""
-wh_sirius_pointshop#0,"""Visiphone / Idola Point Exchange"""
-wh_myset#0,"""Counters / My Set"""
-wh_team_shop#0,"""Counters / Team Shop"""
-wh_equip_note#0,"""Counters / Collection File"""
-wh_battle_counter#0,"""Counters / Battle Counter"""
-wh_petrabo#0,"""Shops / Pet Lab"""
-wh_challengemile_shop#0,"""Shops / Challenge Shop"""
-wh_treasure_shop#0,"""Shops / Treasure Shop"""
-wh_challenge_ranking#0,"""Counters / Challenge Ranking"""
-wh_arena_ranking#0,"""Terminals / Battle Ranking"""
-wh_stylecounter#0,"""Terminals / Style Counter"""
-wh_craft_pa_custom#0,"""Crafting / PA Customization"""
-wh_am_trade_n#0,"""Casino / Normal Prize Exchange"""
-wh_am_trade_p#0,"""Casino / Limited Prize Exchange"""
-wh_battlecoin_shop#0,"""Shops / Battle Coin Exchange Shop"""
-wh_questcounter_mltpt#0,"""Counters / Quest Counter"""
-wh_mltpt_supplement#0,"""Counters / Multi-Block Matching"""
-wh_tl_co_021283_v#0,"""PSO2es Linking"""
+wh_pet#0,"""Inventory/Pet"""
+wh_equip_note_menu#0,"""Inventory/Collection File"""
+wh_equip_accessory#0,"""Equip Menu/Accessories"""
+wh_pollute_form_list#0,"""Equip Menu/Dark Blast"""
+wh_pollute_skill_list#0,"""Equip Menu/Dark Blast Skills"""
+wh_equip_fashion#0,"""Equip/Fashion"""
+wh_fashioncat#0,"""Equipment/Fashion Catalog"""
+wh_fashioncat_ranking#0,"""Equipment/View Everyone's Fashion"""
+wh_fashioncat_search#0,"""Equipment/Search Everyone's Fashion"""
+wh_fashioncat_register#0,"""Equipment/Register Your Fashion"""
+wh_fashioncat_confirm#0,"""Equipment/Check Registered Fashion"""
+wh_fashioncat_profile#0,"""Equipment/Fashion Information"""
+wh_favfashion#0,"""Equipment/Bookmarks"""
+wh_storyboard_questrank#0,"""Quest/Story Quest Difficulty"""
+wh_arksleague#0,"""Quests/ARKS League"""
+wh_recommend#0,"""Quests/Featured Quests"""
+wh_campaignquest#0,"""Quests/Campaign Quests"""
+wh_missionboard#0,"""Quests/ARKS Missions"""
+wh_missionpass#0,"""Quests/Mission Pass"""
+wh_sgemshop#0,"""AC Menu/SG Shop"""
+wh_sgshop#0,"""AC Menu/Purchase Star Gems"""
+wh_sirius_pointshop#0,"""Visiphone/Idola Point Exchange"""
+wh_myset#0,"""Counters/My Set"""
+wh_team_shop#0,"""Counters/Team Shop"""
+wh_equip_note#0,"""Counters/Collection File"""
+wh_battle_counter#0,"""Counters/Battle Counter"""
+wh_petrabo#0,"""Shops/Pet Lab"""
+wh_challengemile_shop#0,"""Shops/Challenge Shop"""
+wh_treasure_shop#0,"""Shops/Treasure Shop"""
+wh_challenge_ranking#0,"""Counters/Challenge Ranking"""
+wh_arena_ranking#0,"""Terminals/Battle Ranking"""
+wh_stylecounter#0,"""Terminals/Style Counter"""
+wh_craft_pa_custom#0,"""Crafting/PA Customization"""
+wh_am_trade_n#0,"""Casino/Normal Prize Exchange"""
+wh_am_trade_p#0,"""Casino/Limited Prize Exchange"""
+wh_battlecoin_shop#0,"""Shops/Battle Coin Exchange Shop"""
+wh_questcounter_mltpt#0,"""Counters/Quest Counter"""
+wh_mltpt_supplement#0,"""Counters/Multi-Block Matching"""
```

Comment:
For these, it might be more appropriate to replace them with colons, since they're being used as categories rather than alternatives.